docs/pages/application-access/cloud-apis/aws-console.mdx

```diff
@@ -443,6 +443,28 @@ instance and attach an IAM role to it.
 
 (!docs/pages/includes/aws-no-credential-provider.mdx service="Application"!)
 
+### `the read limit is reached` error when updating AWS lambda function
+
+Teleport enforces a 10MB size limit for each HTTP request's body.
+You may encounter this error when updating an AWS lambda function using
+`tsh aws update-function-code --function-name myfunction --zip-file ./file.zip`
+if `file.zip` is larger than 10MB.
+
+AWS also limits zip file size to 50MB.
```

Comment:
If AWS' limit is 50MB, can we make ours match?

Reply:
security pushed back on doing that, because 10MB is already very lenient. The security concern is preventing a Teleport user from doing a resource exhaustion attack by spamming large requests

Reply:
cc @jentfoo